prompts/few_shots.go

```diff
@@ -0,0 +1,22 @@
+package prompts
+
+type FewShotPromptTemplateOption struct {
```

Comment:
Please comment these structures as appropriate

Reply:
This is still a WIP, I will add them later.